README.md

```diff
@@ -20,5 +21,111 @@ FLARE is currently a pure Python framework which doesn't need build. Several dep
+## use a simple MLP network to compute the policy and value
+from flare.model_zoo.simple_models import SimpleModelAC
+## RL agent with simple control logic
+from flare.agent_zoo.simple_rl_agents import SimpleRLAgent
+## the agent collects data in an on-line manner without a replay buffer
+from flare.framework.agent import OnPolicyHelper
+## import a wrapped version of OpenAI Gym
+from flare.framework.env import GymEnv
+```
+
+2. Then we configure the problem and initialize the environments.
+
+```python
+game = "CartPole-v0"
+
+num_agents = 16    ## 16 agents running in parallel
+num_games = 8000   ## each agent runs for 8000 games
+
+envs = []
+for _ in range(num_agents):
+    envs.append(GymEnv(game))  ## init an environment for each agent
+
+## get the environment observation dimension and the number of actions
+state_shape = envs[-1].observation_space.shape[0]
+num_actions = envs[-1].action_space.n
+```
+
+3. Next we construct the neural network and choose the algorithm to train the network.
+
+```python
+## For this simple problem, our model uses an MLP to directly map
+## each input vector to a latent embedding of 128 dimensions.
+mlp = nn.Sequential(
+    nn.Linear(state_shape, 128),
+    nn.ReLU(),
+    nn.Linear(128, 128), nn.ReLU(), nn.Linear(128, 128), nn.ReLU())
+
+## The learning algorithm is specified through defining an Algorithm object,
+## which takes a Model object as its argument. In this example, we uses an
+## algorithm called SimpleAC; it implements simple Actor-critic by taking
+## the latent embedding output by the model defined above to generate a discrete
+## policy and a value.
+alg = SimpleAC(model=SimpleModelAC(
+    dims=state_shape, num_actions=num_actions, mlp=mlp))
+```
+
+4. In this example, each agent has a single computation task (i.e., 'RL'). Below we set the hyperparameters and options for this computation task, and create a Manager from the settings.
+
+```python
+ct_settings = {
+    "RL": dict(
+        algorithm=alg,       ## pass in the algorithm object
+        hyperparas=dict(lr=5e-5),   ## set the learning rate
+        # sampling
+        agent_helper=OnPolicyHelper,   ## the training data collection is on-policy
+        # each agent will call `learn()` every `sample_interval` steps
+        sample_interval=2,
+        num_agents=num_agents)
+}
+### create a manager that handles the running of the whole pipeline
+manager = Manager(ct_settings)
+```
+
+5. Add all the 16 agents to the manager and start training!
+
+```python
+# Spawn one agent for each instance of environment.
+for env in envs:
+    agent = SimpleRLAgent(env, num_games)
+    # An agent has to be added into the manager before we can use it to
+    # interact with environment and collect data
+    manager.add_agent(agent)
+
+manager.start()
+```
+
+Suppose that the root directory of FLARE is `<flare_root>`. To run the above code, you can simply do
+```bash
+cd <flare_root>;
+python flare/examples/ac_example.py
+```
```

Comment:
It would be useful to add a sentence describing what will be expected in the next few minutes after kicking off training (e.g. ~200 average reward in ~X minutes on a machine with Y GPU).

Reply:
added